src/Eurofurence.App.Server.Services/Events/EventService.cs

```diff
@@ -164,93 +167,103 @@ public async Task RunImportAsync()
 
                 if (csvRecords.Count == 0) return;
 
+                var internalTrackNamesLowerCase = _eventOptions.InternalTracksLowerCase;
```

Comment:
Maybe a null check is missing here.
If the entry is not in the config file a null pointer could be thrown.
```suggestion
                var internalTrackNamesLowerCase = _eventOptions.InternalTracksLowerCase ?? Array.Empty<string>();
```